src/main/java/org/apache/commons/compress/archivers/zip/X5455_ExtendedTimestamp.java

```diff
@@ -130,16 +155,20 @@ private static Date zipLongToDate(final ZipLong unixTime) {
         return unixTime != null ? new Date(unixTime.getIntValue() * 1000L) : null;
     }
 
+    private static FileTime zipLongToFileTime(final ZipLong unixTime) {
```

Comment:
The "zipLong" method name prefix is redundant with the argument type `zipLongToFileTime` -> `toFileTime` or probably better `zipLongToFileTime` -> `unixToFileTime`

Reply:
I get it, but it became somewhat confusing having all method be overloaded like that. I can change, though. It's up to you.